### ISSUE TITLE
Enhance AggregationFuzzer to test on plans for global aggregation on an empty input

### ISSUE THREAD

**Issue by aditi-pandit** [facebookincubator/velox#7029]
### Description

Such plans return the default value of the aggregate. This is NULL for most aggregates. But some aggregates do return a reasonable value e.g. count will return 0.

The fuzzer test would ensure that all aggregates can handle such queries.

**Comment by kgpai** [comment]
@aditi-pandit Can we  write a unit test to handle this ? We can get all the aggregation function signatures , create an empty row vector for input type and test it out. One can have a default mapping as a part of this unit test , which any new aggregate  would need to update. 

I feel fuzzer is wrong test here since we arent fuzzing anything here and just validating behavior and thus a unit test seems more appropriate.

**Comment by aditi-pandit** [comment]
@kgpai : I agree this is not a traditional fuzzer test as there really isn't any fuzzing here. 

Though at a high level, it seems to fit in the line of the testing which is the intent of the fuzzer.

The reason this combination of empty input matters for each function is to check if the default value it returns for empty input is consistent with Presto. So it fits somewhere into the line of testing you'll are doing. The general feeling was that we could avoid redoing a similar setup by adding this code here.

Do you still feel that we should separate this test ?
